### PR TITLE
[AMS-2021] Initial event generation

### PR DESCRIPTION
**Pull request by PaulusTM**
Adding initial details for the AMS 2021 event with the team from last year.
Team might change as we progress and event details will follow. 

For now just a date reservation and general announcement the event will take place in 2021.

//cc: @yvovandoorn && @FloorD

### CODE REVIEW

File: data/events/2021-amsterdam.yml [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+description: "Devopsdays is coming to Amsterdam!" # Edit this to suit your preferences
+ga_tracking_id: "" # If you have your own Google Analytics tracking ID, enter it here. Example: "UA-74738648-1"
+
+# All dates are in unquoted 2021-MM-DDTHH:MM:SS+TZ:TZ, like this:
+#   variable: 2019-01-04T00:00:00+02:00
+#   variable: 2019-01-05T23:59:59+02:00
+# Note: we allow 2021-MM-DD for backward compatibility, but it can lead to unexpected behaviors (like your event disappearing from the front page during your last day)
+
+startdate: 2021-06-29T00:00:00+02:00 # The start date of your event. Leave blank if you don't have a venue reserved yet.
+enddate: 2021-06-29T00:00:00+02:00 # The end date of your event. Leave blank if you don't have a venue reserved yet.
```
Comment (yvovandoorn) [inline data/events/2021-amsterdam.yml:14]: You probably need to modify this @PaulusTM to reflect an actual day vs instantaneously starting/stopping.

File: data/events/2021-amsterdam.yml [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+    twitter: "FloorDrees"
+  - name: "Daniela Resch"
+    linkedin: "https://www.linkedin.com/in/danielaresch/"
+  - name: "Walter Heck"
+    linkedin: "https://www.linkedin.com/in/walterheck/"
+    twitter: "walterheck"
+  - name: "Joep Piscaer"
+    linkedin: "https://www.linkedin.com/in/jpiscaer"
+    twitter: "jpiscaer"
+  - name: "Jos van Schouten"
```
Comment (yvovandoorn) [inline data/events/2021-amsterdam.yml:76]: I'm not seeing Fabian here @PaulusTM . Double check with the group to make sure this is exhaustive.

**Comment by yvovandoorn** [comment]
Left a few comments behind @PaulusTM. Once you've addressed them, I'll :shipit: :-).

**Comment by mattstratton** [comment]
@PaulusTM can you review @yvovandoorn 's comments? Thanks!

File: data/events/2021-amsterdam.yml (hunk 1 above)
Comment (PaulusTM) [reply data/events/2021-amsterdam.yml]: Updated to initial placeholder times until program becomes clear.

File: data/events/2021-amsterdam.yml (hunk 2 above)
Comment (PaulusTM) [reply data/events/2021-amsterdam.yml:76]: Add everyone I missed.

**Pull request merged**